samples/clipboard/clipboard.cpp

```diff
@@ -191,6 +191,7 @@ void MyFrame::OnUpdateUI(wxUpdateUIEvent&event)
 
 void MyFrame::OnQuit(wxCommandEvent& WXUNUSED(event))
 {
+    wxTheClipboard->Flush();
```

Comment:
It would be better to add a button or a menu item for flushing the clipboard explicitly. This could also log the success or failure of this operation.